src/main/scala/scala/tools/refactoring/implementations/oimports/ImportsOrganizer.scala

```diff
@@ -0,0 +1,79 @@
+package scala.tools.refactoring
+package implementations.oimports
+
+import scala.tools.nsc.Global
+
+abstract class ImportsOrganizers[G <: Global, U <: TreeToolbox[G]](val treeToolbox: U) {
```

Comment:
outer class defines path dependent type Global and inner Global.Tree but let me check if T can be parameterized like `T <: treeToolbox.global.Tree`